src/stores/activityStore/index.js

```diff
@@ -1,10 +1,30 @@
-import { observable, action } from 'mobx'
+import { action, observable } from 'mobx'
 
+/**
+ *
+ * @param    {boolean}  hasReported       card has already been reported by current user, and can't be modified.
+ * @param    {boolean}  hasforbiddened    card has been over reported and can't do anything else by current user.
+ *
+ */
+
+const activityDataList = [{
```

Comment:
Better to put such mock data in an external file. This history will be kept in git and that's a little awkward.

Reply:
mock data could be set here(https://easy-mock.com)

Reply:
> mock data could be set here(https://easy-mock.com)

OK, I will try it.

Reply:
> Better to put such mock data in an external file. This history will be kept in git and that's a little awkward.

Sure, I will fix it.